src/components/panel/panel.spec.js

```diff
@@ -650,6 +650,83 @@ describe('$mdPanel', function() {
     });
   });
 
+  describe('animation logic', function() {
+    var config;
+    var mdPanelAnimation;
+    var myButton;
+
+    beforeEach(function() {
+      config = {
+        template: '<div>Hello World!</div>'
+      };
+
+      mdPanelAnimation = $mdPanel.newPanelAnimation();
+
+      myButton = '<button>myButton</button>';
+      attachToBody(myButton);
+      myButton = angular.element(document.querySelector('button'));
+    });
+
+    describe('should openFrom', function() {
```

Comment:
nit: This doesn't sound like a sentence. I think you're missing a word somewhere:

"MdPanel animation logic should openFrom provided a selector"